tests/integration/multi_cluster_deploy_test.go

```diff
@@ -13,7 +14,7 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
-func TestInstallingMultipleRookCluster(t *testing.T) {
+func TestMultipleClusterSuite(t *testing.T) {
```

Comment:
In general I've found it best to have the main entry point to running the suite match the same naming scheme as the Suite itself.  It makes it less confusing when you are trying to run just a single suite from the command line.

Reply:
agreed, i'll make it match